test/foundry/deploy/03_Erc20Enforcer.t.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.13;
+
+import {Test, Vm, console} from "forge-std/Test.sol";
+import {Constants} from "../utils/Constants.sol";
+import {HolographERC20} from "../../../src/enforcer/HolographERC20.sol";
+
+contract Erc20Enforcer is Test {
+  uint256 localHostFork;
+  string LOCALHOST_RPC_URL = vm.envString("LOCALHOST_RPC_URL");
+  HolographERC20 holographERC20;
+  string tokenName = "Holograph ERC20 Token";
+  string tokenSymbol = "HolographERC20";
+  uint16 tokenDecimals = 18;
+
+  function setUp() public {
+    localHostFork = vm.createFork(LOCALHOST_RPC_URL);
+    vm.selectFork(localHostFork);
+    holographERC20 = HolographERC20(payable(Constants.getHolographERC20()));
```

Comment:
suggestion: let's add a TODO here, to remember that this should actually be the sampleERC20 address, if I'm not mistaken.

Reply:
probably prefer to delete this file so it is not mixed in this PR and in another one where all this test is developed.